index.bs

```diff
@@ -485,6 +521,155 @@ spec:csp3; type:grammar; text:base64-value
   failed resource with a different one.
 
   <!-- ####################################################################### -->
+  
+  ## Integrity-Policy ## {#integrity-policy-section}
+  The `Integrity-Policy` and `Integrity-Policy-Report-Only` HTTP headers enable a document to
+  enforce a policy regarding the integrity metadata requirements on all the subresources it
+  loads of certain <a for=request>destinations</a>.
+
+  The headers' value is a <a>Dictionary</a> [[RFC9651]], with every member-value being an
+  <a>inner list</a> of <a>tokens</a>.
+  
+  A <dfn>source</dfn> is a string. The only possible value for it is "inline".
+  A <dfn>destination</dfn> is a string. The only possible value for it is "script".
+
+  An <dfn>integrity policy struct</dfn>, is a <a>struct</a> that contains the following:
+
+    * <dfn>sources</dfn>, a list of <a>source</a>s, Initially empty.
+    * <dfn>blocked destinations</dfn>, a list of <a>destination</a>s, Initially empty.
+    * <dfn>endpoints</dfn>, a <a>list</a> of strings, initially empty.
+
+  When <dfn>processing an integrity policy</dfn>, with |input|, do the following:
+
+  1. Let |integrity policy| be a new <a>integrity policy struct</a>.
+  1. Let |dictionary| be the result of <a>parsing a dictionary</a> with |input|.
+  1. If |dictionary|['sources'] does not exist or if its value <a for=list>contains</a> "inline",
+     <a for=list>append</a> "inline" to |integrity policy|'s <a>sources</a>.
+  1. If |dictionary|['blocked-destinations'] exists:
+    1. If its value <a for=list>contains</a> "script",
+       <a for=list>append</a> "script" to |integrity policy|'s <a>blocked destinations</a>.
+  1. if |dictionary|['endpoints'] exists:
+    1. Set |integrity policy|'s <a>endpoints</a> to |dictionary|['endpoints'].
+  1. return |integrity policy|
```

Comment:
1. Return should not be lowercase.
2. You miss a trailing dot.
3. Variables should be camelcase for better readability (to distinguish one from two variables).